pkgs/development/libraries/libgit2/default.nix

```diff
@@ -11,6 +11,7 @@
 , http-parser
 , libiconv
 , Security
+, static ? stdenv.hostPlatform.isStatic
```

Comment:
Do you think there's value in having on option for static-only builds outside of `pkgsStatic`?  What's the use case?

Reply:
@alyssais 

Actually, I'd like to use this on my pkg: https://github.com/NixOS/nixpkgs/pull/178258

The build, however, failed because libgit2 provides no static library.

https://logs.nix.ci/?key=nixos/nixpkgs.178258&attempt_id=d45207c8-b5ae-4d4f-98a0-ec8740cbfecc

The way to provide static libs is the same with OpenSSL.

https://github.com/NixOS/nixpkgs/pull/178258/files#diff-9ef181cfa53a0525427ee77a47e88808db67f1d6108870ab58c6c3db89a3f021R116

Reply:
Okay, makes sense.